aten/src/ATen/native/cpu/ActivationKernel.cpp

```diff
@@ -0,0 +1,484 @@
+#include "ATen/native/cpu/ActivationKernel.h"
+
+#include <algorithm>
+#include <iterator>
+#include <limits>
+#include <numeric>
+
+#include "ATen/Dispatch.h"
+#include "ATen/Parallel.h"
+#include "ATen/cpu/vec256/vec256.h"
+#include "ATen/optional.h"
+
+// NOTE: In general we avoid calls into cmath for code compiled with AVX/AVX2
+// This is because of SSE-AVX transitions and a bug in Glibc2.23
+// See https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1663280
```

Comment:
Ok, is it specific to any part of this file? It currently appears to be out of context. Can you make it `[Note AVX-SSE transitions]` and later have a comment saying `// See [Note AVX-SSE transitions]` when it's relevant?

Reply:
Sure.